### PR TITLE
SSE-2872:Logging visibility to be updated when service name not regis…

### PR DESCRIPTION
**Pull request by DStumpGDS**
Scenario: when service registration failed while creating an account 

relevant user cannot able to access the service and the user is not getting an option to either to create a service(ideal way) or not getting access to service which is displaying while logging in to application
stepFunction: RegisterAuthClient

### CODE REVIEW

File: express/src/controllers/register.ts [hunk 1]
```diff
@@ -394,3 +405,11 @@ export const resumeUserJourneyAfterPassword: RequestHandler = async (req, res) =
     // This is to allow for the fact that they may have entered an incorrect number.
     return res.redirect("enter-phone-number");
 };
+
+export const rolbackServiceEntriesForClient: RequestHandler = async (req, res) => {
```
Comment (MartinH-DI) [inline express/src/controllers/register.ts:409]: rolbackServiceEntriesForClient -> rollbackServiceEntriesForClient ??

**Comment by DStumpGDS** [comment]
Good spot Martin! - Method name now changed to correct one.

File: express/src/controllers/register.ts (hunk 1 above)
Comment (DStumpGDS) [reply express/src/controllers/register.ts]: Good spot Martin! - Method now changed.

**Review by MartinH-DI — DISMISSED** [review]
Looks good

File: express/src/controllers/register.ts [hunk 2]
```diff
@@ -333,9 +333,20 @@ export const processAddServiceForm: RequestHandler = async (req, res, next) => {
...
+    try {
+        console.info("Generating Client to Service:" + service.serviceName);
+        const generatedClient = await s4.generateClient(service, nonNull(req.session.authenticationResult));
+        body = JSON.parse(generatedClient.data.output).body;
+        serviceId = JSON.parse(body).pk;
+    } catch (error) {
+        console.error("Unable to Register Client to Service - Service Items removed");
+        console.error(error);
+        await s4.deleteServiceEntries(uuid);
+        return res.render("there-is-a-problem.njk");
```
Comment (rafal-surowiec-co) [inline express/src/controllers/register.ts:348]: return res.render("rollback-service-entries.njk");

**Review by rafal-surowiec-co — CHANGES_REQUESTED** [review]

File: express/src/controllers/register.ts (hunk 2 above)
Comment (DStumpGDS) [reply express/src/controllers/register.ts:348]: Hi Rafal, I think the actual solution is not what you suggested but to remove the rollbackServiceEntriesForClient Request Handler and Routes associated with it - I think this was originally put in to do the Logic of removing the Entries but resulted in just redirecting to the Problem Page which I now do in the Catch Section anyway (the Handler is effectively redundant).

Comment (DStumpGDS) [reply express/src/controllers/register.ts:348]: Rafal has agreed to removal of redundant Request Handler and associated Route.

**Review by rafal-surowiec-co — APPROVED** [review]

**Pull request merged**